adagrams/game.py

```diff
@@ -1,11 +1,142 @@
+import random
+import copy
+
+ # Two Dimensional Array
+
+LETTER_POOL = [
+	['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A'],
+	['B', 'B'], 
+	['C', 'C'], 
+	['D','D','D','D'],
+	['E','E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'], 
+	['F', 'F'], 
+	['G', 'G', 'G'], 
+	['H', 'H'], 
+	['I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I'], 
+	['J'], 
+	['K'], 
+	['L', 'L', 'L', 'L'], 
+	['M', 'M'],
+	['N', 'N', 'N', 'N', 'N', 'N'],
+	['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O'], 
+	['P', 'P'], 
+	['Q'], 
+	['R', 'R', 'R', 'R', 'R', 'R'], 
+	['S', 'S', 'S', 'S'], 
+	['T', 'T', 'T', 'T', 'T', 'T'], 
+	['U', 'U', 'U', 'U'], 
+	['V', 'V'], 
+	['W', 'W'], 
+	['X'], 
+	['Y', 'Y'],
+	['Z']
+]
+
+# helper function to create one big list of strings
+
+def get_one_list_of_letters():
```

Comment:
Yessss! I love how you are making use of helper functions here! It allows how us make more concise and readable code! Also great job on leaving a comment about what this function does.